api/v0/ingest/schema/schema.ipldsch

```diff
@@ -29,9 +57,14 @@ type Advertisement struct {
     #  * https://ipld.io/specs/advanced-data-layouts/hamt/spec/#use-as-a-set
     Entries Link
     # ContextID is the unique identifier for the collection of advertised multihashes.
+    # If a Provider listing is written with no ContextID and IsRm=false, peers from ExtendedProvider 
+    # will be returned for all advertisements published by the publisher.
```

Comment:
I assume that the indexer's list is updated every time there is a provider listing with no context ID, and that the indexer only returns the latest version of the list of providers, regardless of what CID is queried. Is that correct?

Reply:
The indexer's ExtendedProviders list is updated every time when it ingests a new ad with ExtendedProviders. Providers from that list will be returned either for all cids (previous and future ones) if the ad didn't have contextID or only for a specific context id (if the ad had the context ID set). It can also be a union of both but that would require 2 different ads one with contextID one without.

Reply:
Will add a link to the spec. Otherwise I'll have to copy and paste it's contents.